Makefile

```diff
@@ -116,13 +116,12 @@ else
 endif
 
 CPP        := cpp
-ELF2ROM    := tools/buildtools/elf2rom
 MKLDSCRIPT := tools/buildtools/mkldscript
 MKDMADATA  := tools/buildtools/mkdmadata
-YAZ0       := tools/buildtools/yaz0
 ZAPD       := tools/ZAPD/ZAPD.out
 FADO       := tools/fado/fado.elf
-MAKEYAR    := tools/buildtools/makeyar
+MAKEYAR    := python3 tools/buildtools/makeyar.py
```

Comment:
Perhaps better in a different PR, but since you are adding some new python scripts, could you make a `PYTHON` variable (like you do in oot) `PYTHON ?= python3`. Would probably put it in defaults above since I think we want it over writable.

Side note; Would you also be able to fix the `#MIPS toolchain` typo and add a new line between the defaults and `#### Setup ####`